firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -45,7 +45,7 @@ class BaseTestCase: XCTestCase {
         // Send app to background, and re-enter
         XCUIDevice.shared.press(.home)
         // Let's be sure the app is backgrounded
-        _ = app.wait(for: XCUIApplication.State.runningBackgroundSuspended, timeout: TIMEOUT_LONG)
```

Comment:
I would not change this.. there might be other tests that need to wait a longer time..

Reply:
i  can revert that, but to me 45 seconds is to long to wait

Reply:
let's try and see....